arkane/statmech.py

```diff
@@ -320,17 +323,31 @@ def create_log(log_path):
         except KeyError:
             raise InputError('Required attribute "energy" not found in species file {0!r}.'.format(path))
         if isinstance(energy, dict):
-            energy = {k.lower(): v for k, v in energy.items()}  # Make model chemistries lower-case
+            # Standardize model chemistry names
+            energy = {standardize_name(k) if isinstance(k, str) else k: v for k, v in energy.items()}
+            freq_level = getattr(self.level_of_theory, 'freq', self.level_of_theory)
```

Comment:
Does `self.level_of_theory` get assigned to anything other than `None` before this point?